HesoyamHospital/PharmacyRegistration/Repository/SQLRepository/Base/MyDbContext.cs

```diff
@@ -32,12 +32,16 @@ private bool isPostgres()
             return Environment.GetEnvironmentVariable("USES_POSTGRES") == "TRUE";
         }
 
-
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<RegisteredPharmacy>().OwnsOne(rp => rp.ApiKey);
+            modelBuilder.Entity<RegisteredPharmacy>().OwnsOne(rp => rp.Endpoint);
+        }
 
         private string GenerateConnectionString()
         {
             string server = Environment.GetEnvironmentVariable("DATABASE_HOST") ?? "localhost";
-            return "server=" + server.Trim() + ";" + Environment.GetEnvironmentVariable("MyDbConnectionString");
+            return "server=" + server.Trim() + ";" + "port = 3306; database = mydb1; user = root; password = root";
```

Comment:
Please return the original version of the generate method, or at least surround it with an IF statement, because completely hardcoding the connection string will disable it from working on deployment platforms (Docker/Heroku).
You can add a new Environment Variable that specifies whether to choose one method or another, or you could copy this version of the connection string and paste it in launchSettings.json (I recommend the latter).